RCSwitch.h

```diff
@@ -125,6 +125,7 @@ class RCSwitch {
         HighLow syncFactor;
         HighLow zero;
         HighLow one;
+        boolean bLowFirst;
```

Comment:
Hmm, if you instead called this `boolean invertHighAndLow`, or just `boolean invert`, and then put the high value into the low field, and vice versa, wouldn't this avoid most of the changes in `receiveProtocol`? Personally, I'd find that easier to grasp, as that is really all that happens: the meanings of high and low are swapped.

Alas, talk is cheap, work is hard: You patch works (as you are using it), I am just hypothesizing... :-)